### ISSUE TITLE
Obsolete `Main.gd`?

### ISSUE THREAD

**Issue by nonchip** [godotrecipes/2d_grid_movement#3]
the `Main.gd` script (on `Main.tscn:root`) declares a draw method that seemingly does nothing visible on screen, i think that got in the repository on accident?

**Comment by cbscribe** [comment]
Yep, that was from an earlier iteration of the example before adding tile images.
